library/nametab.ml

```diff
@@ -450,19 +450,7 @@ let push_xref visibility sp xref =
     | Until _ ->
         the_ccitab := ExtRefTab.push visibility sp xref !the_ccitab;
         the_globrevtab := Globnames.ExtRefMap.add xref sp !the_globrevtab
-    | _ ->
-        begin
-          if ExtRefTab.exists sp !the_ccitab then
-            let open GlobRef in
-            match ExtRefTab.find sp !the_ccitab with
-              | TrueGlobal( ConstRef _) | TrueGlobal( IndRef _) |
-                    TrueGlobal( ConstructRef _) as xref ->
```

Comment:
It's not the same if this `as xref` gets bound to a different `xref` than the one we got as argument, but I don't know if that can ever happen and I don't know what it would mean.